docs/pages/includes/role-spec.mdx

```diff
@@ -392,7 +392,9 @@ spec:
     # access_plugin_data - allows modifying Access Request plugin data
     #
     # session            - session playback records
-    # ssh_session        - an active SSH session
+    # session_tracker    - an active session
+    # ssh_session        - allows seeing active sessions page
```

Comment:
We should probably just deprecate this, since the active sessions page now shows all types of sessions.

Reply:
Should we go ahead with this update and put a GH issue for `ssh_session`? I left it in another page since if you deny it you won't see the active sessions page.

Reply:
@zmb3 is there an objection to updating this now?